tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/MatterProvision.cs

```diff
@@ -254,9 +254,88 @@ public GenericResponseVM UpdateMatterMetadata(MatterMetdataVM matterMetadata)
                 Dictionary<string, string> propertyList = SetStampProperty(client, matter, matterDetails);
                 matterRepositoy.SetPropertBagValuesForList(clientContext, matterStampedProperties, matter.Name, propertyList);
                 //As part of final step in matter creation, check whether any assigned users are external to the 
-                //organization and if yes, send notification to that user to accepct the
-                //inviotation so that he can access matter center
-                externalSharing.ShareMatter(matterInfo);
+                //organization and if yes, send notification to that user to accepct the invitation so that he can access matter center
+
+                //Need to loop each matter information, update the table storage with that matter information, before sending
+                //notification to external user
+                if (matterInfo.Matter != null && matterInfo.Matter.Roles != null && matterInfo.Matter.Roles.Count > 0)

```

Comment:
When sharing the matter for multiple external users, the matter information is getting stored in the azure table storage. 
When storing the matter information in Azure table storage, for each external user associated to the matter, we need to have an individual row in the azure table storage
Made changes to the code to support that implementation